Classes/Console/Error/ErrorHandler.php

```diff
@@ -52,6 +52,50 @@ public function setExceptionalErrors(int $exceptionalErrors)
         $this->exceptionalErrors = $exceptionalErrors;
     }
 
+    /**
+     * Converts error level int to friendly error type text
+     *
+     * @param int $errorLevel The error level - one of the E_* constants
+     * @return string Error level as text
+     */
+    public function errorLevelToText(int $errorLevel): string
+    {
+        switch ($errorLevel) {
+            case E_ERROR: // 1 //
+                return 'E_ERROR';
```

Comment:
The intention of the transformation was to create human readable labels out of error codes. This isn't the case any more with this change.

Reply:
I've included the old labels and edit the new ones to fit the same style.